PRIVACY.md

```diff
@@ -0,0 +1,21 @@
+**Last updated**  
+January 27 2023
+
+# Privacy Policy
+Catima does not collect or transmit any personal information.
+
+To ensure correct app functionality, we require access to the following:
+
+- Camera: We need access to your camera to be able to scan barcodes. The app can still be used when camera access is denied, but you will have to manually type the barcode information.
+- Storage (Android 5 and 6 only): We need access to your device storage to create or import backups. The app can still be used when storage access is denied, but you will not be able to create or import backups.
+
+Catima offers a feature to share cards with other users. All the relevant data is in the generated shareable URLs and never transmitted to our servers. When viewed through catima.app, the data in the URL is rendered using client-side Javascript to further ensure no data is ever transmitted to us.
+
+# Changes
+This Privacy Policy may be updated from time to time for any reason. We will notify you of any changes to our Privacy Policy by posting the new Privacy Policy on the Catima website. You can view the Privacy Policy as of the time that version of the app was published directly in the app itself, though this may not reflect later changes to the online version. You are advised to consult this Privacy Policy regularly for any changes, as continued use is deemed approval of all changes. You can check the history of this policy by checking this history of this file in GitHub.
```

Comment:
```suggestion
This Privacy Policy may be updated from time to time for any reason. We will notify you of any changes to our Privacy Policy by posting the new Privacy Policy to https://catima.app/privacy-policy/. A snapshot of the Privacy Policy is available within the Catima app, however, it may be outdated. When the Privacy Policy on the website and in the app differ, the website should be considered leading. You are advised to consult the Privacy Policy regularly for any changes, as continued use is deemed approval of all changes.
```

(Also don't think "You can check the history of this policy by checking this history of this file in GitHub." actually means anything to... most users)

Reply:
I'm not entirely sure how to handle it. And Google may disagree either way.

To me, it seems odd to have the online version be leading as the app is completely offline and any changes are likely to reflect changes to the app itself, unless a mistake was corrected.

The version shipped with the app should reflect what the code of the app itself does. And that should be the only thing that's relevant. There is no "backend" that could have changed.

Otherwise agreed.

Reply:
I also really dislike stuff like "You are advised to consult the Privacy Policy regularly for any changes, as continued use is deemed approval of all changes", but not sure how to handle it better. I guess we could inform users of updates now that it's embedded in the app. But that's out of scope either way.

Reply:
Honestly, the privacy policy literally comes down to "I collect as little data as I possible can". It's a formality, I wouldn't even have written one if not for Google requiring it.

The main goal of the privacy policy is to satisfy Google and cover myself. And it just seems safer to be able to update stuff online, I don't want to end up in a situation where I have to rush out an app update (and risk breaking things) just for a privacy policy fix of any kind.

If this was a real company, yeah, this would irk me too. But right now it's doing the most of preventing drama with a very limited resource: my spare time.

Reply:
Ack. I wanted to point that out, but I certainly see no need to spend a lot of time on this. I will update with your suggested changes.